ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -21,7 +21,7 @@
       :setShowCertifiedCheckbox="false"
       @proceed="onStaffPaymentChanges($event)"
     />
-    <div v-if="dataLoaded && !dataLoadError" class="container pa-0" style="min-width: 960px;">
+    <div v-if="appReady" class="container pa-0" style="min-width: 960px;">
```

Comment:
Since we don't need to load data anymore, because it was loaded in the previous step, I just added a check to ensure the app is ready to deal with instances where the url might be manipulated directly.